Assets/LeapMotion/Scripts/Utils/RuntimeGizmoManager.cs

```diff
@@ -524,13 +524,60 @@ public class RuntimeGizmoDrawer {
     /// <summary>
     /// Draws a wire gizmo circle at the given position, with the given normal and radius.
     /// </summary>
-    public void DrawWireCirlce(Vector3 center, Vector3 direction, float radius) {
+    public void DrawWireCircle(Vector3 center, Vector3 direction, float radius) {
       PushMatrix();
       matrix = Matrix4x4.TRS(center, Quaternion.LookRotation(direction), new Vector3(1, 1, 0)) * matrix;
       DrawWireSphere(Vector3.zero, radius);
       PopMatrix();
     }
 
+    /// <summary>
+    /// Draws a wire gizmo capsule at the given position, with the given start and end points and radius.
+    /// </summary>
+    public void DrawWireCapsule(Vector3 start, Vector3 end, float radius) {
+      Vector3 up = (end - start).normalized * radius;
+      Vector3 forward = Vector3.Slerp(up, -up, 0.5F);
+      Vector3 right = Vector3.Cross(up, forward).normalized * radius;
+
+      float height = (start - end).magnitude;
+      float distToCap = Mathf.Max(0, (height * 0.5F) - radius);
+      Vector3 middle = (end + start) * 0.5F;
+
+      start = middle + ((start - middle).normalized * distToCap);
```

Comment:
Might be controversial, but I think that the 'start' and 'end' params should be the actual points that define the capsule, and not the 'top' and 'bottom' concepts.  Maybe there could be a method overload that allows you to choose the method?